README.md

```diff
@@ -72,8 +72,9 @@ yarn add \
   @near-wallet-selector/neth \
   @near-wallet-selector/xdefi \
   @near-wallet-selector/ramper-wallet \
-  @near-wallet-selector/near-mobile-wallet  \
```

Comment:
What happened to `near-mobile-wallet` in this list?

Reply:
Fixed.